tests/functional/harris/harris_2d.py

```
@@ -1,50 +1,50 @@
 #!/usr/bin/env python3
+
 import os
 import numpy as np
+import matplotlib as mpl
```

Ruff (0.11.9)

`matplotlib` imported but unused

Remove unused import: `matplotlib`

(F401)